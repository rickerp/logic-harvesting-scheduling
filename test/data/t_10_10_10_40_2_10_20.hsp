20
2
2 4 8 5 3 2 4 8 6 4 6 5 7 1 10 3 5 10 8 3
4 2 3 6 10
2 1 4
6 1 4 5 6 11 12
3 2 3 7
4 3 6 10 11
6 1 3 5 7 8 19
6 4 6 9 17 18 19
3 6 9 12
6 7 8 12 17 18 19
4 1 5 11 13
5 3 5 10 12 16
7 3 8 9 11 16 17 18
2 10 14
2 13 15
2 14 16
7 11 12 15 17 18 19 20
7 7 9 12 16 18 19 20
7 7 9 12 16 17 19 20
7 6 7 9 16 17 18 20
4 16 17 18 19
8 7 6 7 10 4 8 5 6 3 6 5 8 5 5 4 1 8 9 7
9 9 5 4 2 5 10 3 1 7 9 10 3 7 7 5 10 6 1 5
40
