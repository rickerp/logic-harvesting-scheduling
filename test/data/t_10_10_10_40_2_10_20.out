100
7 1 4 5 8 13 15 19
6 2 7 10 12 14 20
7 3 6 9 11 16 17 18
